polkadot/runtime/parachains/src/disputes.rs

```diff
@@ -1016,6 +1016,8 @@ impl<T: Config> Pallet<T> {
 					statement,
 					signature,
 				) {
+					log::warn!("Failed to check dispute signature");
```

Comment:
We don't seem to be using a target in the logs from the runtime. The resulting logs look like this, which seems fine to me.
```
WARN tokio-runtime-worker polkadot_runtime_parachains::disputes:
```

That works in the runtime because we are using a different crate for the log primitives and that look like this
```
    ($lvl:expr, $($arg:tt)+) => ($crate::log!(target: $crate::__private_api::module_path!(), $lvl, $($arg)+));
```

... so you get the target by default.